sort-selection/README.md

```diff
@@ -1,8 +1,8 @@
-# Sort Selection
+# Sort a delimited list
 
-This plugin adds the "Sort selection" and "Sort selection with semicolon" commands.
+This plugin adds the "Sort with a punctation mark" command.
```

Comment:
```suggestion
This plugin adds the "Sort delimited list" command.
```

Also please update the name in the code so that the command name matches the project name